torch/csrc/dynamo/eval_frame.c

```diff
@@ -542,10 +549,20 @@ inline static PyObject* eval_custom_code(
     Py_XINCREF(fastlocals_old[nlocals_old + i]);
     fastlocals_new[nlocals_new + i] = fastlocals_old[nlocals_old + i];
   }
+
+  #endif
+
+  result = eval_frame_default(tstate, shadow, throw_flag);
+
+  #if IS_PYTHON_3_11_PLUS
+ cleanup:
```

Comment:
I know CPython uses a lot of gotos but we usually don't. There are 0 usage of goto across torch/ aten/ c10/ today.
I think it is ok to duplicate these 3 cleanup lines above in this case.